modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -29,48 +34,60 @@ You cannot add a local registry to any version of an existing Runtime Fabric ins
 
 * You also need to synchronize in your registry the Mule runtime versions you intend to use for application deployments. Refer to the xref:release-notes::runtime-fabric/runtime-fabric-runtimes-release-notes.adoc[Runtime Fabric Patch Update Release Notes] for information about the latest Mule runtime patch versions. The location of the image will follow this format: `rtf-runtime-registry.kqa.msap.io/mulesoft/poseidon-runtime-4.4.0:<tag>`.
 
-== Configure a Local Registry for Use with Runtime Fabric 
+[[synchronize]]
+== Synchronize Dependencies Images to Your Local Registry
 
-To configure a local registry, you must gather and add the necessary credentials to synchronize the registries. 
-
-=== Before You Begin
+To install Runtime Fabric, you must first synchronize the dependencies images to your local registry. You can perform the synchronization by running a script automatically or running comands manually. 
 
-Ensure that you've performed the following tasks: 
+[tabs]
+====
+Synchronize Automatically:: 
 
-. Set up, configure, and test your private Docker image registry.
-. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
-+
-See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version. 
 +
+--
+To synchronize the dependencies images automatically, you ran a bash script that uses `docker`, `curl`,  and `jq` tools.
+
 [NOTE]
-Dependency versions are specific to the Runtime Fabric version.
- 
+Automated registry synchronization is available from Runtime Fabric version 2.6.0 onwards. Previous dependencies can not be retrieved using this script.
 
-. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Download and install the `rtfctl` command line utility] version 0.3.150 or higher.
-. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Create a Runtime Fabric using Runtime Manager], and gather the activation data. 
+Ensure that your xref:access-management::permissions-by-product.adoc#runtime-manager[Access Management permission for Runtime Manager] is set to *Read Runtime Fabric*, which enables you to query Runtime Fabrics in your organization.
 
-=== Configure Your Local Registry and Install Runtime Fabric
+. In your shell script, run the following script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
 
-. Obtain `rtf-runtime-registry` credentials:
 +
 [source,copy]
 ----
-# rtfctl get registry-credentials <activation-data>
+
+##!/usr/bin/env bash
+dependencies=$(curl https://qax.anypoint.mulesoft.com/runtimefabric/api/agentmanifests/<agent-version>         -H 'Authorization: bearer <authorization-token>' | jq -c '.dependencies |.[] | select(.provider | contains("generic")) | "\(.artifact):\(.version)"' | sed 's/"//g')
```

Comment:
Where it says `qax.anypoint.mulesoft.com` we should advertise it as the platform url, and tell customers which values they can use.